test/upgrade-tests.sh

```diff
@@ -20,8 +20,21 @@ failed=0
 (( !failed )) && install_catalogsource || failed=3
 (( !failed )) && logger.success 'Cluster prepared for testing.'
 
-(( !failed )) && install_serverless_previous || failed=5
-(( !failed )) && run_knative_serving_rolling_upgrade_tests || failed=6
+# Test upgrades from CSV one version back, with manual approval.
+(( !failed )) && install_serverless_previous || failed=4
+(( !failed )) && run_knative_serving_rolling_upgrade_tests || failed=5
+(( !failed )) && teardown_serverless || failed=6
+
+# Test upgrades from CSV two versions back to the latest, with automatic approval.
```

Comment:
Do we need to test both `from two versions back` and `from oldest`? I wonder that the `from oldest` covers `from two versions back`.

Reply:
Yeah. I think these two tests are different and we need them both. If you install something at version X and upgrade to latest, it is different from installing it at version X+1 and upgrading to latest. Some issues from the past remind me that upgrading from X to X+1 is not same as installing it freshly at version X+1.